script.js

```diff
@@ -1,4 +1,116 @@
+var rollDice = function () {
+  var randomDecimal = Math.random() * 6;
+  var randomInteger = Math.floor(randomDecimal);
+  var diceNumber = randomInteger + 1;
+  return diceNumber;
+};
+
+// initialise game mode
+var currentGameMode = "choose order of dice";
+var user = "Player 1";
+var diceNumberPlayer1 = 0;
+var diceNumberPlayer2 = 0;
+var diceRoll1 = 0;
+var diceRoll2 = 0;
+
+var whoWins = function (diceNumberPlayer1, diceNumberPlayer2) {
+  if (diceNumberPlayer1 > diceNumberPlayer2) {
+    return "Player 1 wins. Click submit to play again";
+  } else if (diceNumberPlayer2 > diceNumberPlayer1) {
+    return "Player 2 wins. Click submit to play again";
+  } else if (diceNumberPlayer1 == diceNumberPlayer2) {
+    return "It is a draw. Click submit to play again";
+  }
+};
+
+var diceRollSame = function (user, diceRoll1, diceRoll2) {
+  return `Welcome ${user}. <br> You rolled ${diceRoll1} for Dice 1 and ${diceRoll2} for Dice 2. <br> Your number is ${diceRoll1}${diceRoll2}.`;
+};
+
+var diceRollDiff = function (user, diceRoll1, diceRoll2) {
+  return `Welcome ${user}. <br> You rolled ${diceRoll1} for Dice 1 and ${diceRoll2} for Dice 2. Choose the order of the dice.`;
+};
```

Comment:
good try at using functions for outputing messages, this could be combined together, putting the 'if' statement inside to check if the dicerolls are the same